crates/stark-felt/src/lib.rs

```diff
@@ -224,8 +307,117 @@ impl TryFrom<&Felt> for NonZeroFelt {
     }
 }
 
+impl From<usize> for Felt {
```

Comment:
Shouldn't we use this trait for those functions ? https://docs.rs/num/latest/num/trait.FromPrimitive.html

Reply:
I'm not entirely sure about introducing the dependency, so I'd rather not decide it on my own.
Also, I'm not sure the reason, but it used to be part of the stdlib and got deprecated.
That said, there's probably a way to make it generic for all primitive types by doing something like:
```rust
impl From<T: From<i128>> for Felt {
    fn from(value: T) -> Self {
        let value: i128 = value.into();
        Self(FieldElement::from(value))
    }
}
```
What do you think?

Reply:
Sounds fine to me,  i'd also even implement the zero/one trait but let's wait for others to review